RecoEgamma/EgammaElectronAlgos/src/SiStripElectronAlgo.cc

```diff
@@ -1149,7 +1149,7 @@ bool SiStripElectronAlgo::projectPhiBand(float chargeHypothesis, const reco::Sup
   double reducedChi2 = (totalNumberOfHits > 2 ? chi2 / (totalNumberOfHits - 2) : 1e10);
 
   // Select this candidate if it passes minHits_ and maxReducedChi2_ cuts
-  if (totalNumberOfHits >= minHits_  &&  reducedChi2 <= maxReducedChi2_) {
+  if (innerhit != nullptr && totalNumberOfHits >= minHits_  &&  reducedChi2 <= maxReducedChi2_) {
```

Comment:
@lgray: Do we really want an exception here? I wouldn't want to stop all processing just because some inner hit is null, unless that is absolutely necessary.